### PR TITLE
bugfix for #72 OBS site search issue

### PR DESCRIPTION
**Pull request by sbatchelder**
#72 OBS site search issue

Some udf FieldValue.field_value fields were in the database as `None` and could not be rendered in the table. Changing `FieldValue.__str__()` to return `""` when `FieldValue.field_value==None` fixes this bug. 

The issue was not sorting related, it only so-happened that the UDF's that needed rendering (even in a hidden column) made it to the paginated table after that particular sort.

### CODE REVIEW

**Comment by ethanandrews** [comment]
Did you run the `manage.py makemigrations` command for the DB change you made? I don't see a migration file as part of this commit.

File: roundabout/userdefinedfields/models.py [hunk 1]
```diff
@@ -49,7 +49,7 @@ def __str__(self):
 
 
 class FieldValue(models.Model):
-    field_value = models.TextField(null=True, blank=True)
+    field_value = models.TextField(null=False, blank=True)
```
Comment (ethanandrews) [inline roundabout/userdefinedfields/models.py:52]: Let's revert this change to the Model field back to the original version. If we try to do a database migration like this, it will fail as the current fields that are Null will now violate this constraint. To do this, we need to first update existing data (either with another django migration or by shell), then change the schema.

**Comment by sbatchelder** [comment]
reverted back to `null=True`
